app/app.go

```diff
@@ -301,12 +316,14 @@ func NewLinkApp(
 		ibctransfertypes.StoreKey,
 		icahosttypes.StoreKey,
 		wasmplustypes.StoreKey,
+		fswaptypes.StoreKey,
+		fbridgetypes.StoreKey,
 	)
 
 	tkeys := sdk.NewTransientStoreKeys(paramstypes.TStoreKey)
 	// NOTE: The testingkey is just mounted for testing purposes. Actual applications should
 	// not include this key.
-	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey)
+	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey, fbridgetypes.MemStoreKey, "testingkey")
```

Comment:
When is "testingkey" used? If "testingkey" is only used for testing, can't it be removed from PROD binary and only used in the test app?